index.html

```diff
@@ -0,0 +1,96 @@
+    <title>Trial Project</title>
+</head>
+<body>
+    <ul class="nav justify-content-end">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="#">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Work</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">Contact</a>
+        </li>
+      </ul>
+    
+      <section class="section-1">
+        <div class="content1">
+          <h1 id="h1-1">Hey We are Donard & Zaliha</h1>
+          <h4 id="h4-1">Software Developers</h4>
+        </div>
+      </section>
+    
+      <section class="section-2">
+        <h3>There are some of my projects</h3>
+    
+        <div class="row row-cols-1 row-cols-md-3 g-4">
+          <div class="col">
+            <div class="card">
+              <img src="images/normanborlaug.jpeg" class="card-img-top" alt="..." />
+              <div class="card-body">
+                <h6 class="card-title">Tribute Page</h6>
+              </div>
+            </div>
+          </div>
+          <div class="col">
+            <div class="card">
+              <img src="images/random-quote-machine.png" class="card-img-top" alt="..." />
+              <div class="card-body">
+                <h6 class="card-title">Random Quote Machine</h6>
+              </div>
+            </div>
+          </div>
+          <div class="col">
+            <div class="card">
+              <img src="images/calculator.png" class="card-img-top" alt="..." />
+              <div class="card-body">
+                <h6 class="card-title">Javascript Calculator</h6>
+              </div>
+            </div>
+          </div>
+          <div class="col">
+            <div class="card">
+              <img src="images/mapacross.png" class="card-img-top" alt="..." />
+              <div class="card-body">
+                <h6 class="card-titler">Map Data Accross The Globe</h6>
+              </div>
+            </div>
+          </div>
+          <div class="col">
+            <div class="card">
+              <img src="images/wikipedia.png" class="card-img-top" alt="..." />
+              <div class="card-body">
+                <h6 class="card-title">Tic Toe Game</h6>
+              </div>
+            </div>
+          </div>
+          <div class="col">
+            <div class="card">
+              <img src="images/tictactoe.png" class="card-img-top" alt="..." />
+              <div class="card-body">
+                <h6 class="card-title">Card title</h6>
+              </div>
+            </div>
+          </div>
+    
+          <button type="button" class="btn btn-primary">Show all ></button>
+        </div>
+      </section>
+    
+      <section class="section-3">
+        <h1>Let's work together...</h1>
+        <p><i>How do you take your coffe?</i></p>
+      </section>
```

Comment:
You did a really great job but please if you can add icons that redirect customers to your social media to know you better and see your work it would be better.